observability/otel/otel-collector/otel-collector-extensions.adoc

```diff
@@ -446,18 +445,35 @@ include::snippets/technology-preview.adoc[]
         protocols:
           http: {}
     exporters:
-      otlp:
+      debug:
 
     service:
       extensions: [zpages]
       pipelines:
         traces:
           receivers: [otlp]
-          exporters: [otlp]
+          exporters: [debug]
 # ...
 ----
 
-<1> Specifies the HTTP endpoint that serves zPages. Use `localhost:` to make it available only locally, or `":"` to make it available on all network interfaces. The default is `localhost:55679`.
+<1> Specify the HTTP endpoint for serving the zPages extension. The default is `localhost:55679`.
+
+The {OTELOperator} does not expose this route. To access the HTTP endpoint, forward the port:
```

Comment:
This to me reads like it should be in an admonition block. 

[NOTE]
====
TEXT
====